chain/epoch-manager/src/lib.rs

```diff
@@ -574,9 +574,9 @@ impl EpochManager {
 
         // Kickout unstaked validators.
         for (account_id, proposal) in all_proposals {
-            if proposal.stake() == Balance::ZERO
-                && *next_epoch_info.stake_change().get(&account_id).unwrap_or(&Balance::ZERO)
-                    != Balance::ZERO
+            if proposal.stake().is_zero()
+                && !(*next_epoch_info.stake_change().get(&account_id).unwrap_or(&Balance::ZERO))
+                    .is_zero()
```

Comment:
There are a couple instances of unnecessary dereference like this that then forces you to add parentheses.

This and other such instances likely could be `!next_epoch_info.stake_change().get(&account_id).unwrap_or(&Balance::ZERO).is_zero()`.